src/openloco/localisation/string_ids.h

```diff
@@ -1316,4 +1352,24 @@ namespace openloco::string_ids
 
     constexpr string_id menu_quit_to_menu = 2139;
     constexpr string_id menu_exit_openloco = 2140;
+
+    constexpr string_id getVehicleType(VehicleType type)
```

Comment:
I think this function should be moved elsewhere. Perhaps a vehicle-related file?